test/e2e-test-application/e2e/tests/0-fiddle/fiddle-navigation.spec.js

```diff
@@ -905,4 +905,37 @@ describe('Fiddle', () => {
       cy.get('.lui-nav-title .fd-nested-list__title').should('contain', 'test');
     });
   });
+
+  describe('Custom text in the footer', () => {
+    context('Desktop', () => {
```

Comment:
There is actually no need to define a 'desktop context' if there is no other context distinction.